lib/round.rb

```diff
@@ -0,0 +1,47 @@
+class Round
+    attr_reader :deck
+    attr_reader :turns
```

Comment:
Instead of having these on two lines, you can put them on the same line by separating the symbols by columns. `attr_reader :deck, :turns`